src/config/alwaysOnLibraries.js

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs';
+import path from 'path';
+
+const alwaysOnLibraries = {
```

Comment:
The reason I went with the separate file versus an attribute was not having to deal with all of the complexity of handling it in the picker, etc. Seemed like this is likely to be a one-off?